### PR TITLE
DOCS-1642: Rename xArmLite to lite6

### PR DESCRIPTION
**Pull request by sguequierre**
Aligning with product name

### CODE REVIEW

File: docs/components/arm/lite6.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Configure an xArmLite Arm"
```
Comment (npentrel) [inline docs/components/arm/lite6.md:2]: remember the alias

**Review by npentrel — APPROVED** [review]
LGTM once you add the alias

File: docs/components/arm/lite6.md (hunk 1 above)
Comment (sguequierre) [reply docs/components/arm/lite6.md:2]: Alias is already present, right? @npentrel am I looking at the right thing?

Comment (npentrel) [reply docs/components/arm/lite6.md:2]: Oh true. Alright then

**Review by npentrel — APPROVED** [review]

**Pull request merged**